test/language/computed-property-names/computed-property-names-prototype.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    computed property names can be "prototype"
+---*/
+class C {
+  static ['prototype']() {
```

Comment:
Yeah, seemed backward to me as well, but... https://github.com/v8/v8-git-mirror/blob/bd21d72d507625e5caf70f31ee11c8c21ead1651/test/mjsunit/harmony/computed-property-names-classes.js#L314-L390